wordpress-mobile/WordPress-iOS
# [Stats] show Posting Activity bottom line

Fixes #12648 

To test:
- Go to Stats > Insights.
- If necessary, add the 'Posting Activity' stat.
- Verify the bottom line appears on the card.

<img width="400" alt="posting_activity" src="https://user-images.githubusercontent.com/1816888/66949655-20b11880-f014-11e9-95a3-71e6d0f47462.png">


Update release notes:
